modules/auxiliary/scanner/winrm/winrm_cmd.rb

```diff
@@ -82,7 +83,8 @@ def run_host(ip)
         framework_module: self,
         cache_file: datastore['WinrmKrb5Ccname'].blank? ? nil : datastore['WinrmKrb5Ccname'],
         mutual_auth: true,
-        use_gss_checksum: true
+        use_gss_checksum: true,
+        ticket_storage: ticket_storage
```

Comment:
```suggestion
        ticket_storage: kerberos_ticket_storage
```

Maybe? Just to namespace things / make it more clear to the user